### PR TITLE
Remove non-functioning servers, add nostrcheck.me, use NIP96 endpoint for all servers

### PR DESCRIPTION
**Pull request by quentintaranpino**
Tests here:

https://nosta.me/nprofile1qyghwumn8ghj7mn0wd68ytnvv9hxgtcppemhxue69uhkummn9ekx7mp0qqsvuwhtzx4a9cqc79nx4cfe7lg0v9q29smhuywtxlv6er8gdl6pt5qn9f7uy

### CODE REVIEW

**Comment by jb55** [comment]
please squash your changes!

**Comment by quentintaranpino** [comment]
Done!

**Comment by jb55** [comment]
Reviewed-by: William Casarin ***@***.***>

**Comment by jb55** [comment]
thanks!

File: damus/Views/AttachMediaUtility.swift [hunk 1]
```diff
@@ -40,7 +40,7 @@ func create_upload_request(mediaToUpload: MediaUpload, mediaUploader: MediaUploa
     request.setValue("multipart/form-data; boundary=\(boundary)", forHTTPHeaderField: "Content-Type")
     
     // If uploading to a media host that support NIP-98 authorization, add the header
-    if mediaUploader == .nostrBuild,
+    if mediaUploader == .nostrBuild || mediaUploader == .nostrcheck,
```
[the remark this answers is not in the document]
Comment (quentintaranpino) [reply damus/Views/AttachMediaUtility.swift:43]: I left it this way thinking about providers outside of nostr. For example nostur uploads images to imgur. But my first thought was the same, all with NIP98.

**Pull request merged**